src/pages/index.js

```diff
@@ -64,7 +64,10 @@ export async function getServerSideProps(context) {
     membersArr.sort((a, b) => (a.first_name > b.first_name ? 1 : -1));
 
     const newMembersArr = membersArray.filter(
-      (person) => !person.isMember && person.first_name !== undefined
+      (person) =>
+        !person.isMember &&
+        person.first_name !== undefined &&
+        !person.archived_member
```

Comment:
Why are we not keeping the field names consistent with camelCasing?

Reply:
I didn't paid much attention, Kill keep in mind from now.